.pipelines/onebranch/pipeline.bootstrapper.pullrequest.yml

```diff
@@ -13,23 +13,26 @@ pr: none
 
 variables:
   Cdp_Definition_Build_Count: $[counter('', 0)] # needed for onebranch.pipeline.version task https://aka.ms/obpipelines/versioning
-  LinuxContainerImage: cdpxlinux.azurecr.io/global/ubuntu-1804-all:5.0   # Docker image which is used to build the project https://aka.ms/obpipelines/containers
+  LinuxContainerImage: cdpxlinux.azurecr.io/global/ubuntu-1804-all:5.0 # Docker image which is used to build the project https://aka.ms/obpipelines/containers
   Debian_Frontend: noninteractive
 
 parameters:
-- name: buildFluentBit
-  displayName: Build Fluent Bit?
-  type: boolean
+  - name: buildFluentBit
+    displayName: Build Fluent Bit?
+    type: boolean
 
 resources:
   repositories:
-  - repository: templates
-    type: git
-    name: OneBranch.Pipelines/GovernedTemplates
-    ref: refs/heads/main
-  - repository: rhado
-    type: git
-    name: ARO.Pipelines
+    - repository: templates
+      type: git
+      name: OneBranch.Pipelines/GovernedTemplates
+      ref: refs/heads/main
+    - repository: rhado
+      type: git
+      name: ARO.Pipelines
+    - repository: rpconfig
+      type: git
+      name: RP-Config
```

Comment:
nit: I think the editor on save may have added white space to this.  Same nit as above with the whitespace.  
_note: still valid yaml_

Reply:
@troy0820 I'll take care of this shortly, thanks!